scripts/libWebGL/yyCommandBuilder.js

```diff
@@ -189,6 +191,7 @@ function yyCommandBuilder(_interpolatePixels) {
 	    m_lastTexture = [];
 	    m_lastVBuffer = undefined;
 	    m_lastShader = undefined;
+        m_scissor = { x : 0, y : 0, w : 0, h : 0 };

```

Comment:
If you don't set a scissor elsewhere is this not going to be leaving you with a 0x0 scissor rect?

Reply:
No as the scissor is set when the viewport is set up....